scripts/run-ci-tests.sh

```diff
@@ -77,11 +77,11 @@ else
                 ;;
             *continuous*)
                 TEST_CONFIG="Continuous"
-		CTEST_OPTIONS+=" --timeout 360"
+		CTEST_OPTIONS+=" --timeout 600"
```

Comment:
We should try to get this down in the future

Reply:
This also needs to be large enough for the -O1 sanitizer runs. It will likely depend on https://github.com/GMLC-TDC/HELICS-src/issues/583 -- figuring out what tests are redundant, maybe testing the functionality of individual cores instead of using all of them for complete system tests.